jans-cedarling/flask-sidecar/main/config.py

```diff
@@ -35,36 +37,44 @@ class BaseConfig:
     API_SPEC_OPTIONS = {
         "x-internal-id": "1",
     }
-    CEDARLING_BOOTSTRAP_CONFIG_FILE = os.getenv("CEDARLING_BOOTSTRAP_CONFIG_FILE", None)
+    CEDARLING_BOOTSTRAP_CONFIG_FILE = os.getenv(
+        "CEDARLING_BOOTSTRAP_CONFIG_FILE", None)
     if CEDARLING_BOOTSTRAP_CONFIG_FILE is None:
-        logger.warning("Cedarling bootstrap file not found")
-        exit()
-    with open(CEDARLING_BOOTSTRAP_CONFIG_FILE, "r") as f:
-        CEDARLING_BOOTSTRAP_CONFIG = f.read()
+        logger.warning(
```

Comment:
`CEDARLING_BOOTSTRAP_CONFIG` may be an unbound variable here. The correct way is to initialize it to a null value before the conditional.
```python
CEDARLING_BOOTSTRAP_CONFIG = None
#...
if CEDARLING_BOOTSTRAP_CONFIG_FILE is None:
```